Assets/__Scripts/MapEditor/Hit Sounds/HitSoundVolumeController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Assets.__Scripts.MapEditor.Hit_Sounds
```

Comment:
Looks like this was added in the IDE, not through Unity

To keep consistency with the rest of the project, would you mind removing the namespace? That's about the only I'd like changed with this PR

Reply:
Indeed it was, will do